src/api/Jalasoft.TeamUp.Resumes.API/Startup.cs

```diff
@@ -19,11 +19,9 @@ public override void Configure(IFunctionsHostBuilder builder)
             builder.Services.AddScoped<IHealthService, HealthService>();
             builder.Services.AddScoped<IHealthRepository, HealthRepository>();
             builder.Services.AddScoped<IResumesService, ResumesService>();
-#if DEBUG
-            builder.Services.AddScoped<IRepository<Resume>, ResumeSQLRepository>();
-#else
             builder.Services.AddScoped<IRepository<Resume>, ResumesInMemoryRepository>();
-#endif
```

Comment:
I think you shouldn't make changes to this file, please double check

Reply:
or at least these lines shouldn't be affected

Reply:
Done